home/ivp-planning/user-interface/the-2d-panel.md

```diff
@@ -1,33 +1,29 @@
-| ![](../../../.gitbook/assets/iVP\_icon\_imported\_pdf\_on\_off.jpg)       | Toggles the visibility of the [imported PDF floor plan](../getting-started/importing-pdfs.md)                                                                                                                                                                                                                |
-
-## Navigation in the the 2D-Panel:
-
-To change the visible area, you can move the camera by holding down the **middle mouse button**. The zoom level can be adjusted by using the **mouse wheel**.
+|                                                                            |                                                                                                                                                            |
+| -------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------  |
+| ![](../../../.gitbook/assets/2D_panel/Icon_regular_mode.jpg)               | Switches to a mode that lets you drag and drop machines and objects                                                                                        |
+| ![](../../../.gitbook/assets/2D_panel/Icon_path_mode.jpg)                  | Switches to the [path tool mode](../advanced-tools/path-tool.md) which allows you to draw [measurements](../advanced-tools/path-tool.md#measurements), [fences](../advanced-tools/fence-tool.md) and [rooms](../advanced-tools/the-room-tool.md)                                                                                                                                                                                       |
+| ![](../../../.gitbook/assets/2D_panel/Icon_markup_mode.jpg)                | Switches to the [markup tool mode](../machines/copy-and-delete-objects.md#copy-objects) which allows you to place markups to annotate issues or highlights |
+| ![](../../../.gitbook/assets/2D_panel/Icon_copy.jpg)                       | [Copies ](../machines/copy-and-delete-objects.md#copy-objects)the selected object(s) in a specified direction                                              |
+| ![](../../../.gitbook/assets/2D_panel/Icon_multicopy.jpg)                  | [Multi-copies](../machines/copy-and-delete-objects.md#multi-copy-objects-in-the-2d-panel) the selected object(s) according to definable parameters         |
+| ![](../../../.gitbook/assets/2D_panel/Icon_rotate_right.jpg)               | Rotates the selected object(s) clockwise by a fixed degree (the degree can be adjusted in the [settings panel](settings-panel.md#))                        |
+| ![](../../../.gitbook/assets/2D_panel/Icon_rotate_left.jpg)                | Rotates the selected object(s) anti-clockwise by a fixed degree (the degree can be adjusted in the [settings panel](settings-panel.md#global-settings))    |
```

Comment:
The header does not exist in settings-panel.md#global-settings. This has been like this before, but we could fix it in one go.